src/routes/tools/extreme-precipitation/_SettingsPanel.svelte

```diff
@@ -27,16 +31,24 @@
     thresholdStore,
     thresholdTypeStore,
     returnPeriodStore,
+    aggregateFnStore,
   } from "./_store";
 
   export let learnMoreContent;
 
-  const { durationInfo, indicatorInfo, periodInfo, thresholdTypeInfo } =
-    learnMoreContent;
+  const {
+    durationInfo,
+    indicatorInfo,
+    periodInfo,
+    thresholdTypeInfo,
+    aggregationInfo,
+  } = learnMoreContent;
   const dispatch = createEventDispatcher();
-  const { location } = locationStore;
+  const { location, boundary } = locationStore;
   const { indicator } = indicatorStore;
 
+  $: isPolygonBoundary = $boundary.id === "locagrid" ? false : true;
```

Comment:
I'm also thinking it might be good to have common tool constants for the boundary types, for example:

```js
// src/routes/tools/_common/constants.js
export const BOUNDARY_LOCA_GRID = "locagrid";
export const BOUNDARY_COUNTY = "county";
export const BOUNDARY_WATERSHED = "watershed";
// etc.
```

it might help with preventing misspelling the boundary types, but we don't have to do it right now.